tests/baselines/reference/quickInfoAndSemanticsOnJsPropertyWithAmbientDeclaration.baseline

```diff
@@ -0,0 +1,35 @@
+Syntactic Diagnostics for file '/tests/cases/fourslash/quickInfoAndSemanticsOnJsPropertyWithAmbientDeclaration.ts':
+/test.js(6,5): error TS8009: The 'declare' modifier can only be used in TypeScript files.
+/test.js(6,19): error TS8010: Type annotations can only be used in TypeScript files.
+
+
+==== /test.js (2 errors) ====
+    class Foo {
+        constructor() {
+            this.prop = { };
+        }
+    
+        declare prop: string;
+        ~~~~~~~
+!!! error TS8009: The 'declare' modifier can only be used in TypeScript files.
+                      ~~~~~~
+!!! error TS8010: Type annotations can only be used in TypeScript files.
+        method() {
+            this.prop.foo
+        }
+    }
+
+Semantic Diagnostics for file '/tests/cases/fourslash/quickInfoAndSemanticsOnJsPropertyWithAmbientDeclaration.ts':
+
+
+==== /test.js (0 errors) ====

```

Comment:
Wait. This is wrong.

Reply:
The fact that the errors don't come up consistently, and that baselining doesn't give the same error results, is very suspicious.

Reply:
Yeah. All tests I've had for this have not been compiler tests.